apps/email/js/cards/fld/account_item.html

```diff
@@ -1,4 +1,4 @@
 <a class="fld-account-item">
   <span class="selected-indicator"></span>
-  <span dir="auto" class="fld-account-name"></span>
-</a>
\ No newline at end of file
+  <span class="fld-account-name"></span>
```

Comment:
Actually, I was wrong.
> this is the email adress. dir="auto"' ordirection: ltr` in the css works equally well. 

is only true for old-style ASCII email addresses. But now, they can have arabic letters (actually most of utf-8 chars now) in their local part. In this case, `direction: ltr` and `dir="auto"` does not behave identically, see http://jsfiddle.net/23sw5o2n/.

I still believe the correct form is with `direction: ltr`, but I can NI Ahmed to be sure. (I mean, both are confusing, but one is less than the other imho).

We should use `direction: ltr` only for fields that contains only an email address.

If it contains an url,  both are fine, because the scheme part will make the BiDi algorithm calculates a LTR direction for the whole string anyway (assuming the IANA won't register scheme with non-latin letters).

If it contains other texts not handled by l10n.js (typically user input) then we nearly always want to use `dir="auto"` or `unicode-bidi: -moz-plaintext`. We cannot use the latter everywhere because ellipsis are currently buggy with it (see [bug 883884](https://bugzilla.mozilla.org/show_bug.cgi?id=883884), unlikely to be fixed soon).

TL;DR my current rules of thumb is:
- For string handled by l10n.js, don't do anything, that's l10n.js or loca job.
- For emails (alone in their html element), use `direction: ltr`
- For other string, 
   - don't do anything if we're sure they will always have the same direction as the system one (but this is really rare because if they are not handled by l10n.js, it usually means they're from user input).
   - Use unicode-bidi: -moz-plaintext if you don't need ellipsis (typically for most input fields).
   - Use `dir="auto"` everywhere else.

Does this make sense to you?